infrastructure/terraform/README.md

```diff
@@ -100,16 +114,46 @@ installation.
 
    If you don't have a successful execution from the beginning, re-run until all is deployed successfully.
 
-### Post-Installation Instructions
+## Post-Installation Instructions
+
+Now that you have deployed all assets successfully for the first time in your Google Cloud Project, the data must flow through all components. You can trigger your Cloud Workflow to execute your Dataform workflow at any time, or you wait until the next days when the Cloud Workflow is going to be executed according to your schedule.
+
+There are two components in this solution that requires data for proper installation and functioning. One is the Looker Studio Dashboard, you only deploy the dashboard after you have executed the last step in the Installation Guide successfully. Another is the ML pipeline, the pipelines compilation requires views and tables to be created so that it can read their schema and define which kind of transformations the columns will suffer during the pipeline execution.  
 
-Now that you have deployed all assets successfully for the first time in your Google Cloud Project, the data must flow through all components. 
+To start the data flow you must perform the following tasks:
 
-At this point in time, you have two options: a) trigger your Cloud Workflow to execute your Dataform worflow. b) wait until the next day 
-when the Cloud Workflow is going to execute according to your schedule. Next, you will need to execute the BigQuery stored procedures 
-having the prefix "invoke_backfill_*".
+1. Run the Cloud Workflow
+
+    On the Google Cloud console, navigate to Workflows page. You will see a Workflow named `dataform-prod-incremental`, then under Actions, click on the three dots and `Execute` the Workflow.
```

Comment:
Add a sentence along the lines: If you have a considerable amount of data in your exported GA4 and Ads BigQuery datasets it can take several minutes or hours to process all the data. Make sure that the processing is complete before you continue to the next step.

Reply:
fixed